frontend/services/formatting/formatDate.ts

```diff
@@ -0,0 +1,62 @@
+export const invoiceDate = (input) => {
+	const date: Date = new Date(input);
+	let invoiceDateStr: string;
+
+	invoiceDateStr =
```

Comment:
I have added a small date library on a branch with the migration of teamspace. It's called `dayjs` (https://github.com/iamkun/dayjs). I think you could add this library too and then refactor a bit that file.

Reply:
we decided that we aren't doing it now